File: LeetCode/cpp/convertToTitle.cpp
#include <string>

class Solution {
public:
    std::string convertToTitle(int columnNumber) {
        std::string result;
        while (columnNumber > 0)
        {
            result = getTheBase_26(columnNumber) + result;
        }
        
        return result;
    }

private:
    char getTheBase_26 (int& numero) {
        int resultInt = (numero - 1) % 26;
        numero = (numero - 1) / 26;
        
        return resultInt + 'A';
    }
};

File: LeetCode/go/leetcode/convertToTitle.go
package leetcode

func getTheBase_26(numero *int) (byte) {
	var resultInt int = ((*numero) - 1) % 26
	*numero = (*numero - 1) / 26

	return byte(resultInt + 'A')
}

func ConvertToTitle(columnNumber int) string {
    var result string

	for ; columnNumber != 0 ; {
		result = string(getTheBase_26(&columnNumber)) + result
	}

	return result
}